gulpfile.js

```diff
@@ -70,7 +72,25 @@ gulp.task('test', function(done) {
   new karmaServer(config, done).start();
 });
 
-gulp.task('watch', ['clean:tests', 'build-tests'], function() { gulp.watch(PATHS.src, ['build-tests']); });
+gulp.task('tdd', ['clean-build-tests'], function(done) {
+  var karmaServer = require('karma').Server;
+  var travis = process.env.TRAVIS;
+
+  var config = {configFile: __dirname + '/karma.conf.js', singleRun: false, autoWatch: true};
```

Comment:
I don't think this option parameters are needed. They exist on the karma conf already. (singleRun, autoWatch).